applications/rubintv/templates/deployment-frontend.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "rubintv.fullname" . }}-frontend
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "rubintv.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: "Recreate"
+  template:
+    metadata:
+      {{- with .Values.frontend.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "rubintv.selectorLabels" . | nindent 8 }}
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - name: {{ .Chart.Name }}-frontend
```

Comment:
These don't need to be globally unique, so rather than make a complicated name that would be annoying if we ever had to specify it explicitly with the `-c` option, you can just use `frontend` here.